spec/syntax.md

```diff
@@ -672,13 +611,17 @@ name-char  = name-start / DIGIT / "-" / "." / ":"
            / %xB7 / %x300-36F / %x203F-2040
 ```
 
+> **Note**
+> _External variables_ names are constrained by this namespace.
```

Comment:
Strictly speaking not true. Anything not fitting _name_ just won't be matched by any variable in the message.

Reply:
That's correct. I ran into this issue when drafting this section and it occurred to me that we needed to deal with the fact that `name` is constrained but callers will not be constrained. I'll refactor this sentence to be correct, but it is the formatting spec that needs instructions on handling of external variables, which I recorded in #442.